.github/workflows/native-ci.yml

```
@@ -13,24 +13,102 @@ jobs:
+          hyperfine --version
+
       - name: Run tests with coverage
         run: |
+          set -euo pipefail
+
+          normalize_coverage_paths() {
+            local profile="$1"
+            local module_path="$2"
+            local module_dir="$3"
+            python - "$profile" "$module_path" "$module_dir" <<'PY'
+import pathlib
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)